hardhat.config.js

```diff
@@ -36,7 +39,7 @@ module.exports = {
       {
         version: "0.6.9",
         settings:{
-          optimizer: { enabled: true, runs: 200 }
+          optimizer: { enabled: true, runs: 10000 }
```

Comment:
why 10000?

Reply:
probably done in order to trace hardhat gas results difference